src/peft/tuners/lora/hqq.py

```diff
@@ -0,0 +1,234 @@
+from __future__ import annotations
```

Comment:
Could you please add the license notice to the top of the file, like other files have?